susemanager/etc/logrotate.d/susemanager-tools

```diff
@@ -11,4 +11,14 @@
     su wwwrun www
 }
 
-
+/var/log/rhn/mgr-create-bootstrap-repo/mgr-create-bootstrap-repo.log {
+    compress
+    dateext
+    daily
+    maxage 30
+    missingok
+    notifempty
+    rotate 30
+    size 4096k
+    su wwwrun www
```

Comment:
I think we do not need the "su" option. I think it is more important to give the new directory the correct permissions.
Under which user is mgr-create-bootstrap-repo normally executed?

AFAIK reposync is called via taskomatic which should use root/root. When run manually on the commandline, also root should do it.
No `su` and directory owned by `root` should be fine.
Needs to be tested.

Reply:
More data:

`mgr-create-bootstrap-repo` is called interactively by `root` (assuming the user is using that user to call it) and by reposync https://github.com/uyuni-project/uyuni/blob/master/backend/satellite_tools/reposync.py#L681

reposync, when scheduled from the WebUI, is running as `root`:

```
root     27529  9423 99 20:03 ?        00:03:26 /usr/bin/python3 -u /usr/bin/spacewalk-repo-sync --channel sle-module-basesystem15-sp3-pool-x86_64-sms-4.2 --type yum --non-interactive
```

This means that the directory storing the logs `/var/log/rhn/mgr-create-bootstrap-repo/` is created with `uid=wwwrun` (or `uid=apache` for non-SUSE) and `gid=0` https://github.com/uyuni-project/uyuni/blob/master/backend/common/rhnLog.py#L109-L110:

```
# ls -dl /var/log/mgr-create-bootstrap-repo
drwxr-xr-x 2 wwwrun root 43 Mar  3 20:10 mgr-create-bootstrap-repo
# ls -l /var/log/mgr-create-bootstrap-repo/mgr-create-bootstrap-repo
total 12
-rw-rw---- 1 wwwrun root 9744 Mar  3 20:11 mgr-create-bootstrap-repo.log
```

Test without 'su' option:

```
# date -s 'tomorrow'
# date
Thu  4 Mar 20:15:10 CET 2021
# logrotate /etc/logrotate.conf
# ls -l /var/log/rhn/mgr-create-bootstrap-repo
-rw-rw---- 1 wwwrun root    0 Mar  4 20:15 mgr-create-bootstrap-repo.log
-rw-rw---- 1 wwwrun root 1460 Mar  3 20:14 mgr-create-bootstrap-repo.log-20210304.xz
```